template/docs/_static/css/custom.css

```diff
@@ -13,9 +13,28 @@ html[data-theme="dark"] {
  */
 .bd-header .navbar-nav li a.nav-link .icon-link-image {
     filter: var(--scipp-filter-header-icon);
+    height: 1em;
```

Comment:
This gives the conda logo the same size as the other icons which have shrunk in the latest theme version.